setup.py

```diff
@@ -20,9 +20,18 @@
 # access to a pre-Mavericks mac, so hopefully this won't break on someone's
 # older system.  Tested and it works on Mac OSX Catalina.
 compiler_and_linker_args = []
-if platform.system() == 'Darwin':
-    compiler_and_linker_args = ['-stdlib=libc++']
-
+include_dirs = [numpy.get_include(), 'src/natcap/invest/managed_raster']
+if platform.system() == 'Windows':
+    compiler_args = ['/std:c++20']
+    library_dirs = [f'{os.environ["CONDA_PREFIX"]}/Library/lib']
+    include_dirs.append(f'{os.environ["CONDA_PREFIX"]}/Library/include')
+else:
+    library_dirs = []
+    compiler_args = []
+    compiler_and_linker_args = ['-stdlib=libc++', '-std=c++20']
```

Comment:
Good question, I think I used some C++20 features during development, but now only need C++17. (It still compiles with 11, but I get warnings that the inline variables are a C++17 feature.) Updated in d50a2c9ee

Reply:
Sounds good!  Thanks for looking into it.

If I had to guess, maybe the lack of an `--ffp-contract=off` flag (needed for addressing a compilation bug in viewshed) is causing the rest of the errors?  If (big if) that's the only thing standing in the way, then the viewshed tests would fail, which isn't worth it if all we need is a recent enough version of the C++ standard.